.github/workflows/deploy-pages.yaml

```
@@ -3,7 +3,7 @@ name: Generate docs
 on:
   push:
     branches:
-      - scaladoc # todo change this to the "main" v3 branch
+      - v3-online-interpreter # todo change this to the "main" v3 branch
```

```suggestion
      - version-3
```
Leaving this suggestion here to apply before merging